LEAF_Request_Portal/sources/System.php

```diff
@@ -1109,7 +1118,9 @@ private function processServices(array $portal_services, array $portal_chiefs, a
             } else {
                 // service does not exist add it to the portal db
                 //echo 'The service \'' . $service['service'] . '\' was added.<br/>';
-                $org_service->importService($service['serviceID'], $service['service'], $service['abbreviatedService'], $service['groupID']);
+                if(is_numeric($service['serviceID']) && !empty($service['service']) && (is_numeric($service['groupID']) || is_null($service['groupID']))) {
```

Comment:
You don't need the empty() function here.  is_numeric() covers that.  A null or empty string returns false.